CONTRIBUTING.md

```diff
@@ -1,41 +1,43 @@
 That being said, it also needs to be functional in-game. You could submit a PR based on the grounds that it works, but thorough testing is required to make sure that it doesn't break any other feature.
 
-## Assets
-Though they are fundamental, contributing to these isn't as desired due to how easy it is to contribute this way. Just like Coding, there are a few guidelines that need to be followed.
+## Textures
+Though they are fundamental, contributing to these isn't as desired by developers due to how easy it is to contribute this way. Just like coding, however, there are a few guidelines that need to be followed.
 
 ### Please try to submit the JSON for the texture you wish to add
 If something is implemented but untextured, you may wish to add a texture to those. However, it is most desirable if a JSON is submitted this way too. Unless specified, a PR is unlikely to be pulled if a texture is added but no JSON is attached.
 
-### Regarding JSONS, use Forge Blockstates
+### Use Forge Blockstates for block JSONs
 If you are submitting a PR for a block alongside a JSON, please be sure to use the Forge Blockstate. These reduce on file count and is the desired format. Unless the model does not exist for the block, try to use the existing models from Minecraft.
 
-### Regarding Textures, make sure they are 16*16 pixels
-This might be a mod, but it needs to stay true to Minecraft's art style. Do not exceed 16*16 unless specified by the author.
+### Only make 16x16 pixel textures
+This might be a mod, but it needs to stay true to Minecraft's art style. Do not exceed 16x16 unless specified by the author.
 
-### Regarding Translations, make sure that it is sound quality
-I cannot understand languages outside of English, so if you are ready to have your lang file pulled, please ping the author that you are ready to do so. Lang files should also be formatted as `xx_xx`, as it is the current format.
 
 ### It needs to look good
-This is a no-brainer. To filter out random "contributors" from the mod, the texture needs at least *some* effort put into it. Try not to use third-party editors like Novaskin, as they are limited in tools. Recommended tools are Paint.NET, GIMP and Photoshop.
+This is a no-brainer. To filter out random "contributors" from the mod, the texture needs at least *some* effort put into it. Try not to use third-party editors, such as [Novaskin](https://novaskin.me/), as they are limited in tools. Recommended tools are [Paint.NET](https://www.getpaint.net/), [GIMP](https://www.gimp.org/) and [Photoshop](https://photoshop.com/).
+
+# Localization and Translating
+
+Most contributors cannot understand languages outside of English, so if you are ready to have your `.lang` file pulled, please notify the author that you are ready to do so.
+
+### Naming conventions
+`.lang` files should also be formatted as `xx_xx.lang`, as it is the current format. (Example: `en_US.lang`)
```

Comment:
Example should be `en_us`, not `en_US`. This was a change I believe introduced in either 1.11 or 1.12.